bool    e_stop
float64 velocity_current
float64 relative_theta
float64 front_arm_angle
float64 rear_arm_angle
---
float64 new_velocity
